react/features/filmstrip/components/web/ThumbnailWrapper.js

```diff
@@ -113,26 +113,27 @@ function _mapStateToProps(state, ownProps) {
         const { columns, rows } = gridDimensions;
         const index = (rowIndex * columns) + columnIndex;
         let horizontalOffset;
+        const { iAmRecorder } = state['features/base/config'];
+        const participantsLenght = remoteParticipantsLength + (iAmRecorder ? 0 : 1);
```

Comment:
Is this here +1 or +1 ... I had impression we need to lower the number of participants ....

Reply:
Aaa it was like that before

Reply:
We practically did. then the check is ` participantsLenght-1`.